src/pages/Feed/Feed.jsx

```diff
@@ -44,6 +44,7 @@ export default function Feed() {
 
   return (
     <div id="feed">
+      <h1 id="feed-swipe-instructions">Start swiping left or right</h1>
```

Comment:
I would prefer the current page be displayed at the top of the page (I suspect doing this in the navbar will be a lot of a hassle).

The instructions should be more clear on what each action does. What happens if I swipe left/right?

Reply:
I think we could add a title to every page, so that could be discussed later on. And I'll add the swipe directions.